src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -31,7 +31,8 @@
     public String islandWithNameAlreadyExists = "%prefix% &7An Island with that name already exists.";
     public String islandSchematicNotFound = "%prefix% &7No schematic with that name exists.";
     public String islandDeleted = "%prefix% &7Your island has been deleted.";
-    public String teleportingHome = "%prefix% &7Teleporting to your Island Home";
+    public String teleportingHome = "%prefix% &7Teleporting to your Island Home.";
+    public String teleportingHomeOther = "%prefix% &7Teleporting to %owner%'s Island Home.";
```

Comment:
home should be lowercase

Reply:
I agree. But should Island also be lower case?

Reply:
I think so

Reply:
There are like 100 messages with inconsistent capitalization. There needs to be a definite choice

Reply:
Island should be uppercase. I tried to unify it in the Javadocs as well as the messages but for some reason, I forgot that search + replace is a thing. Just make home lowercase for now :D

Reply:
Peaches probably also added a few new messages with island lowercase